lib/galaxy/tools/parameters/basic.py

```diff
@@ -151,7 +151,7 @@ def value_to_display_text( self, value, app ):
         Convert a value to a text representation suitable for displaying to
         the user
         """
-        if value:
+        if value is not None:
             return unicodify( value )
         return "Not available."
```

Comment:
It was intentional because a value not being available/empty is different then an invalid input. But I can see that more clarity would help here in terms of the display text.